app/forms/register_nonprofit_form/nonprofit_form.rb

```diff
@@ -1,54 +1,51 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
+
 class RegisterNonprofitForm::NonprofitForm < ApplicationForm
   attribute :name, :string, default: "" # needed because OnboardAccounts is goofy
 
-  attr_accessor :city, 
-    :email,
-    :state_code,
-    :phone,
-    :website,
-    :zip_code
+  attr_reader :website
+  attr_accessor :city, :email, :state_code, :phone, :zip_code
```

Comment:
I'd really prefer these on separate lines like they were originally. It significantly improves the `git blame` experience if there are ever changes to the list of attributes.

Reply:
I typically priortize code readability (in its current form, in my editor) over something like historic blame visibility. And for me, long lists where there is one item per row is hard to scan and not as readable.

Also, (for me), it is rare I want to blame which attribute in a list was added or removed (but if I want to, I can figure it out by looking at diffs). 

Additionally git diff in github does a pretty great job of highlighting items that have changed in a list (see attachment)... ![2025-04-08 at 6 42 PM](https://github.com/user-attachments/assets/3b5d1a1a-c618-464a-a229-393e88f81df8)

In any case, my logaf is 2 and ok making the change....